test/kafka_avro_confluent/core_test.clj

```diff
@@ -36,6 +40,23 @@
         (is (sut-reg/get-latest-schema-by-subject schema-registry
                                                   (str topic "-value")))))))
 
+(deftest avro-serde-without-logical-types
```

Comment:
idk if this is really needed, it's just testing the underlying library at this point?

Reply:
Just a regression test to be sure people could still use it the old way. Happy to remove it though :D